### PR TITLE
typesafe v0.2.1-p2

### PR DESCRIPTION
**Pull request by dvirtz**
* I've followed [this guide](https://docs.hunter.sh/en/latest/creating-new/create/cmake.html)
  step by step carefully. **[Yes]**

* I've tested this package remotely and have excluded all broken builds.
  Here is the links to the Travis/AppVeyor with status "All passed":

  * https://ci.appveyor.com/project/dvirtz/hunter/builds/22787251
  * https://travis-ci.org/dvirtz/hunter/jobs/501186108

---

### CODE REVIEW

File: cmake/projects/type_safe/hunter.cmake [hunk 1]
```diff
@@ -57,11 +57,11 @@ hunter_add_version(
     PACKAGE_NAME
     type_safe
     VERSION
-    "0.2.1-p1"
+    "0.2.1-p2"
```
Comment (ruslo) [inline cmake/projects/type_safe/hunter.cmake]: Please create a new entry instead of substituting.

**Comment by dvirtz** [comment]
p1 is broken so I saw no point in leaving it in

**Comment by ruslo** [comment]
> p1 is broken so I saw no point in leaving it in

It's not absolutely broken (see [tests passed](https://github.com/ruslo/hunter/pull/1741#issuecomment-464529155)) and there is no security issue there. However, if the user locked `0.2.1-p1` in a custom config, after URL/SHA1 update of Hunter, the configuration **will be** broken absolutely.

**Comment by ruslo** [comment]
Testing:
- https://ci.appveyor.com/project/ingenue/hunter/builds/22814491
- https://travis-ci.org/ingenue/hunter/builds/501669902?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Uploading:
- https://ci.appveyor.com/project/ingenue/hunter/builds/22814928
- https://travis-ci.org/ingenue/hunter/builds/501676475?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Released: https://github.com/ruslo/hunter/releases/tag/v0.23.126

**Comment by dvirtz** [comment]
Thanks!

**Pull request merged**